ppxlib.opam

```diff
@@ -15,14 +15,16 @@ run-test: [
 ]
 depends: [
   "ocaml"                   {>= "4.04.1"}
-  "base"                    {>= "v0.11.0"}
   "dune"                    {>= "1.11"}
   "ocaml-compiler-libs"     {>= "v0.11.0"}
   "ocaml-migrate-parsetree" {>= "1.5.0"}
   "ppx_derivers"            {>= "1.0"}
-  "stdio"                   {>= "v0.11.0"}
+  "sexplib0"
+  "stdlib-shims"
   "ocamlfind"               {with-test}
-  "cinaps"                  {with-test & >= "v0.12.1"}
```

Comment:
Do you know why the version constraint is not necessary anymore?

Reply:
Not sure. Not even sure it's not required. May have just gotten lost in the shuffle. I'll look into it.

Reply:
In doubt, we might as well keep it. Less chances for things to go wrong.